tests/tools/test_parsing_and_dumping.py

```diff
@@ -14,31 +15,26 @@
 from ssz.sedes import (
     List,
 )
-from ssz.tools import (
-    DefaultCodec,
-    from_formatted_dict,
-    to_formatted_dict,
-)
 
 
 def test_parsing_and_dumping():
-    json_str = json.dumps(to_formatted_dict(zoo))
-    read_zoo = from_formatted_dict(json.loads(json_str), Zoo)
+    json_str = json.dumps(ssz.tools.to_formatted_dict(zoo))
```

Comment:
In general I think this is a pattern we should be avoiding.  It makes grepping for import statements hard and in general goes against the "be explicit" philisophy.  What was wrong with importing the necessary things from `ssz.tools` explicitely?